### PR TITLE
[feat] update to 0.9.19

### PR DESCRIPTION
**Pull request by stanly-johnson**
Changelog:
- Update to 0.9.19
- Tested to build blocks with rococo-local

### CODE REVIEW

File: node/src/command.rs [hunk 1]
```diff
@@ -68,15 +69,15 @@ impl SubstrateCli for Cli {
 
 impl SubstrateCli for RelayChainCli {
 	fn impl_name() -> String {
-		"Virto Parachain Collator".into()
+		"Parachain Collator Template".into()
```
Comment (olanod) [inline node/src/command.rs:72]:
```suggestion
		"Virto Parachain Collator".into()
```

File: node/src/command.rs [hunk 2]
```diff
@@ -68,15 +69,15 @@ impl SubstrateCli for Cli {
...
 	}
 
 	fn impl_version() -> String {
 		env!("SUBSTRATE_CLI_IMPL_VERSION").into()
 	}
 
 	fn description() -> String {
-		"Virto Parachain Collator\n\nThe command-line arguments provided first will be \
+		"Parachain Collator Template\n\nThe command-line arguments provided first will be \
```
Comment (olanod) [inline node/src/command.rs:80]:
```suggestion
		"Virto Parachain Collator\n\nThe command-line arguments provided first will be \
```

File: runtime/Cargo.toml [hunk 3]
```diff
@@ -1,17 +1,17 @@
 [package]
 authors = ['Daniel Olano <daniel@olanod.com>']
 edition = '2021'
-homepage = 'https://github.com/virto-network/virto-node'
+homepage = 'https://github.com/open-web3-stack/virto-node'
```
Comment (olanod) [inline runtime/Cargo.toml:4]:
```suggestion
homepage = 'https://github.com/virto-network/virto-node'
```
Search-replace?

File: runtime/Cargo.toml [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 license = 'Unlicense'
 name = 'virto-runtime'
-repository = 'https://github.com/virto-network/virto-node'
+repository = 'https://github.com/open-web3-stack/virto-node'
```
Comment (olanod) [inline runtime/Cargo.toml:7]:
```suggestion
repository = 'https://github.com/virto-network/virto-node'
```

File: runtime/Cargo.toml [hunk 5]
```diff
@@ -142,7 +142,6 @@ std = [
 	"orml-unknown-tokens/std",
 	"orml-xtokens/std",
 	"virto-primitives/std",
-	"orml-payments/std",
```
Comment (olanod) [inline runtime/Cargo.toml:145]: not needed?

**Review by olanod — DISMISSED** [review]

**Review by olanod — APPROVED** [review]

File: runtime/Cargo.toml (hunk 5 above)
Comment (stanly-johnson) [reply runtime/Cargo.toml:145]: yes

**Review by olanod — APPROVED** [review]

**Pull request merged**